besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -3789,4 +3791,29 @@ public void assertThatAllPortsSpecifiedSucceeds() {
         "--miner-stratum-port=15");
     assertThat(commandErrorOutput.toString()).isEmpty();
   }
+
+  @Test
+  public void quorumInteropEnabledFailsWithoutGasPriceSet() throws IOException {
+    final Path genesisFile = createFakeGenesisFile(GENESIS_QUORUM_INTEROP_ENABLED);
+    parseCommand("--genesis-file", genesisFile.toString());
+    assertThat(commandErrorOutput.toString())
+        .contains(
+            "--min-gas-price must be set to zero if Quorum interop is enabled in the genesis config.");
```

Comment:
here we're calling it Quorum interop. TransactionValidator has "Quorum compatible". I don't have strong feelings but should be consistent.

Reply:
No strong feelings either.  If you need a tie break, I'd choose "GoQuorum compatible"